news/5031.bugfix

```diff
@@ -0,0 +1 @@
+Bug fix for user registration when email is used as username. @mehedikhan72 [#5031](https://github.com/plone/volto/issues/5031)
```

Comment:
For future contributions:

- A commit message should say what it actually fixes, not just refer to an issue number.
- We already know it is a bug fix by its extension of `.bugfix` so saying it is a bug fix in the message is redundant. Towncrier automatically renders sections as shown in this example: https://6.docs.plone.org/volto/release-notes/index.html#alpha-22-2023-07-28
- Your GitHub handle should be the last part of a change log entry.

```suggestion
Allow a user to register when they use an email address as their username. [#5031](https://github.com/plone/volto/issues/5031) @mehedikhan72
```